### PR TITLE
Case bugz-907: On launch default to non hmd device

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.atlassian.net/browse/BUGZ-907

On every launch the app directed by the launcher, the default display is defaulted to desktop mode instead of hmd. 

Test case: 

1.  The app should ignore any settings change for default display and start in desktop mode.
2. rest app, should start in desktop mode
3. go to hmd mode
4. close app with hmd mode on
5. Open app, with launcher and app should open in desktop mode again.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20278/HighFidelity-Beta-PR15899-45b339a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20278/HighFidelity-Beta-Interface-PR15899-45b339a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20278/HighFidelity-Beta-PR15899-45b339a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20278/HighFidelity-Quest-Beta-PR15899-45b339a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20278/HighFidelity-Beta-PR15899-45b339a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20278/HighFidelity-Beta-Interface-PR15899-45b339a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20281/HighFidelity-Beta-PR15899-09ecb49.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20281/HighFidelity-Beta-Interface-PR15899-09ecb49.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20281/HighFidelity-Beta-PR15899-09ecb49.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20281/HighFidelity-Quest-Beta-PR15899-09ecb49.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20281/HighFidelity-Beta-PR15899-09ecb49.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20281/HighFidelity-Beta-Interface-PR15899-09ecb49.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20309/HighFidelity-Beta-PR15899-e463a82.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20309/HighFidelity-Beta-Interface-PR15899-e463a82.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20309/HighFidelity-Beta-PR15899-e463a82.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20309/HighFidelity-Quest-Beta-PR15899-e463a82.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20309/HighFidelity-Beta-PR15899-e463a82.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20309/HighFidelity-Beta-Interface-PR15899-e463a82.exe

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -5406,31 +5406,42 @@ void Application::loadSettings() {
     }
 
     bool isFirstPerson = false;
-    if (_firstRun.get()) {
-        // If this is our first run, and no preferred devices were set, default to
-        // an HMD device if available.
+    if (arguments().contains("--no-launcher")) {
```
Comment (huffman) [inline interface/src/Application.cpp]: Right now we don't have a specific flag that I'm aware of that signifies that we should be in "work" mode. `--no-launcher` currently is only set when in "work" mode, but it doesn't necessarily mean that, so I'm worried about this breaking in the future. I'd be okay merging this if we can create a ticket to add the notion of a "work" mode to support this kind of runtime decision-making.

Comment (huffman) [reply interface/src/Application.cpp]: I talked to Amer, we will add a ticket.

**Pull request merged**